lib/crowbar/client/request/base.rb

```diff
@@ -50,7 +50,7 @@ def params
           when :post
             [
               method,
-              content.to_json
+              content
```

Comment:
This means we will POST the parameters for all the requests (except proposal/edit) url-encoded instead of as JSON again. Is this really desired? I was under the impression that JSON is the preferred encoding.

BTW, why are we using different encodings for POST and PUT?